SprengerV/bike_check
# adding in delete button with admin rendering logic

**Overview**:
Added in delete button to bike posts. The delete button will only render on posts if you are the author or if you have admin permissions. You can find the delete button in the dropdown menu that is shown when you click the "..." on the top right of a post. I was thinking if we end up getting enough time to add edit functionality the edit option could go in that drop down. 

**Changes**:
- Added in font awesome icons
- Refactored DisplayPosts to functional component to match the rest of our components
- Added delete button to posts
- Fixed api issue from my last pr that prevented admins from deleting posts/comments

**How To Test:**
I found out that the easiest way to test this is to login with your normal account and create a post. Then you can log out and create a new account with a fake email and login and see if you can see the delete button. You can also go in the Auth0 console to add admin permissions to that new account after and then you should be able to delete the post your other account created.
